### PR TITLE
bpo-33541: Remove unused __pad function

### PR DESCRIPTION
**Pull request by mariocj89**
The function was added with the initial implementation 00efe7e.

Should be safe to remove as it is private and mangled.

https://bugs.python.org/issue33541

### CODE REVIEW

**Review by grimreaper — APPROVED** [review]

**Comment by encukou** [comment]
I don't think this needs a NEWS entry, but it does have the potential to break some code. It's been around for some time.
Can you create an issue on bugs.python.org, as a place for discussion?

**Comment by mariocj89** [comment]
Thanks @encukou, Created the issue for discussion. The function is mangled so I'd be surprise if anything was using it though

**Comment by encukou** [comment]
I talked to Łukasz, the Python 3.8 release manager, and he said this does need a NEWS entry. I added one. Sorry for the confusion!

**Review by encukou — APPROVED** [review]
I plan to merge this after CI checks finish.

**Pull request merged**
